<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>763</width>
    <height>604</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout" stretch="1,5,1,1">
    <property name="sizeConstraint">
     <enum>QLayout::SetDefaultConstraint</enum>
    </property>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout" stretch="3,0">
      <item>
       <widget class="QGroupBox" name="group_serial">
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>250</height>
         </size>
        </property>
        <property name="title">
         <string>Serial Port</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="4,0,0,0">
           <item>
            <widget class="QComboBox" name="serialPortInfoListBox"/>
           </item>
           <item>
            <widget class="QPushButton" name="button_refreshPorts">
             <property name="text">
              <string>Refresh</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_connect">
             <property name="text">
              <string>Connect</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="button_disconnect">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="text">
              <string>Disconnect</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QLabel" name="descriptionLabel">
           <property name="text">
            <string>Description:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="manufacturerLabel">
           <property name="text">
            <string>Manufacturer:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="serialNumberLabel">
           <property name="text">
            <string>Serial number:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="locationLabel">
           <property name="text">
            <string>Location:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="vidLabel">
           <property name="text">
            <string>Vendor ID:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="pidLabel">
           <property name="text">
            <string>Product ID:</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="group_tests">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="title">
         <string>Tests</string>
        </property>
        <layout class="QGridLayout" name="gridLayout">
         <item row="2" column="0">
          <widget class="QCheckBox" name="checkBoxSaveToFile">
           <property name="text">
            <string>Save To File</string>
           </property>
          </widget>
         </item>
         <item row="1" column="0">
          <widget class="QCheckBox" name="checkBoxContinuous">
           <property name="text">
            <string>Continuous</string>
           </property>
          </widget>
         </item>
         <item row="0" column="0">
          <widget class="QPushButton" name="button_testImage">
           <property name="text">
            <string>Test Camera</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QWidget" name="widget_testImage" native="true"/>
         </item>
         <item row="3" column="0">
          <widget class="QCheckBox" name="checkBoxShowMetadata">
           <property name="text">
            <string>Show Metadata</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QGroupBox" name="group_image">
      <property name="maximumSize">
       <size>
        <width>16777215</width>
        <height>250</height>
       </size>
      </property>
      <property name="title">
       <string>Image Acquisition</string>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <widget class="QLabel" name="label_image_2">
         <property name="text">
          <string>Image</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QProgressBar" name="progressBar_imageAcquisition">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="value">
          <number>0</number>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGroupBox" name="group_power">
      <property name="title">
       <string>Power Consumption</string>
      </property>
      <widget class="QLabel" name="labelPower1">
       <property name="geometry">
        <rect>
         <x>80</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>--</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_2">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Power 0 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_3">
       <property name="geometry">
        <rect>
         <x>220</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>--</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_4">
       <property name="geometry">
        <rect>
         <x>160</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Power 1 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_5">
       <property name="geometry">
        <rect>
         <x>280</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Power 2 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_6">
       <property name="geometry">
        <rect>
         <x>340</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>--</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_7">
       <property name="geometry">
        <rect>
         <x>390</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>Power 3 :</string>
       </property>
      </widget>
      <widget class="QLabel" name="labelPower1_8">
       <property name="geometry">
        <rect>
         <x>450</x>
         <y>20</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>--</string>
       </property>
      </widget>
     </widget>
    </item>
    <item>
     <widget class="QTextBrowser" name="textBrowser">
      <property name="verticalScrollBarPolicy">
       <enum>Qt::ScrollBarAlwaysOn</enum>
      </property>
      <property name="documentTitle">
       <string/>
      </property>
      <property name="lineWrapMode">
       <enum>QTextEdit::WidgetWidth</enum>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>763</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
